sass/_Variables.scss

```diff
@@ -12,6 +12,7 @@ $color-greyish-dark-blue: #263e55;
 $color-greyish-teal-blue: #1d4f76;
 $color-blueish-gray: #67768b;
 $color-vibrant-blue: #009ddc;
+$color-blue-background: #486fc7;
```

Comment:
Could we have re-used another blue already in the project ? And if not, does it pass accessibility contrast ?

Reply:
I tested out the other blues in the project, but unfortunately, they were too ugly as a background. This particular blue felt particularly appropriate as a background color. This blue also has a contrast ratio of 4.8:1 against white.